src/geoms/constants.ts

```diff
@@ -6,4 +6,15 @@ export const colors = {
   copper: [0.9, 0.6, 0.2],
   fr4Green: [0x05 / 255, 0xa3 / 255, 0x2e / 255],
   fr4GreenSolderWithMask: [0x00 / 255, 0x98 / 255, 0x13 / 255],
+  fr1Copper: [0.8, 0.4, 0.2],
+  fr1CopperSolderWithMask: [0.9, 0.6, 0.2],
 } satisfies Record<string, RGB>
+export const boardMaterialColors: Record<string, RGB> = {
+  fr1: colors.fr1Copper,
+  fr4: colors.fr4Green,
+}
+
+export const tracesMaterialColors: Record<string, RGB> = {
```

Comment:
P.S. you can extract the material with `PcbBoard["material"]` or whatever

```suggestion
export const tracesMaterialColors: Record<Material, RGB> = {
```